src/app/_utils/Auth.ts

```diff
@@ -1,5 +1,10 @@
-export default class Auth {
-    static isAuthorized(): boolean{
-        return localStorage.getItem("loggedIn")?.toLocaleLowerCase() === "true";
+import { CanActivate } from "@angular/router";
+
+export default class Auth implements CanActivate {
+    canActivate(): boolean {
```

Comment:
let's dive into CanActivate interface a bit, 

sometimes, we need to check if user is allowed to access the specific content, may I ask you to implement one more guard, this guard will be controlling only route with a specific id, so you will have as
```
  { path: 'admin/:id', component: AdminAPIComponent, canActivate: [Auth, PermissionGuard] },
``` 

the goal behind this guard will be get the id that was provided and run some simple promise to mocked API service ( this part is not that important ), I would like you to understand how we can access the id from inside the guard